plugins/tab/plugin.js

```diff
@@ -65,9 +64,7 @@
 						} else if ( next ) {
 							next = new CKEDITOR.dom.element( next );
 							resultRange.moveToElementEditStart( next );
-							// Avoid selecting empty block makes the cursor blind.
-							if ( !( resultRange.checkStartOfBlock() && resultRange.checkEndOfBlock() ) )
```

Comment:
Is this change necessary? If it's added only to force our fake selection after pressing the <kbd>Tab</kbd> key, I'd remove it. It causes issues in Firefox and IE11 in editors without the `tableselection` plugin. After pressing the <kbd>Tab</kbd> key, the cursor disappears.

Reply:
This change is not necessary. It was just my assumption with false selection. I did not realize that this was causing additional problems. So I will bring back the previous changes.